js/upgrade.js

```diff
@@ -320,14 +320,17 @@ function startProcess(type) {
 }
 
 function afterUpgradeNow(res) {
-  startProcess("upgrade");
-  $("#upgradeNow")
-    .unbind()
-    .replaceWith(
-      "<span id=\"upgradeNow\" class=\"button-autoupgrade\">"
-      + input.translation.upgradingPrestaShop
-      + " ...</span>"
-    );
+  $('#confirm-upgrade').modal()
+    .on('click', '#confirm-upgrade-btn', function(e) {
+      startProcess('upgrade');
+      $("#upgradeNow")
```

Comment:
```suggestion
      $('#upgradeNow')
```

When I said move double quote to single quote, it was for the whole file :p

Reply:
well the file is 800 lines :D I need an automation tool then, looking into it